src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "./Breadcrumb.module.css"
+
+interface LinkProps {
+  href: string
+  isCurrent: boolean
```

Comment:
I'm wondering if `isCurrent` should be a computed property since, as I understand it, the last item in the list is the current page. Is there a scenario that I'm not considering?